src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx

```diff
@@ -70,12 +72,37 @@ const Row = ({ children, Icon, label }) => {
  * @component
  */
 const MissionDetailsCard = ({ mission, photoDisabled }) => {
-  photoDisabled = true;
+  const snackbarContext = useContext(Snackbar.Context.SnackbarContext);
+  const firebase = getFirebase();
+  const storage = firebase.storage();
   const classes = useStyles();
   const [file, setFile] = useState(null);
 
-  function handleImageChosen(file) {
+  if (mission.status === "started") {
+    photoDisabled = false;
+  } else {
+    photoDisabled = true;
+  }
+
+  async function handleImageChosen(file) {
     setFile(file);
+    if (file) {
+      const uploadTask = storage
+        .ref(`confirmed_delivery/${file.name}`)
+        .put(file)
+        .then((data) => {
+          return data.ref.getDownloadURL();
```

Comment:
How do we access this image now?
does this URL need to get added to the mission?

Reply:
Yes I have been trying to add the URL to the mission, but it wont update the firebase for some reason